Firestore/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Unreleased
+- [changed] Firestore now limits the number of concurrent document lookups it
+  will perform when resolving inconsistencies in the local cache (#5310).
```

Comment:
This really should link to the issue rather than the PR--it doesn't matter that the issue was in another repository. See below for a what a link looks like to an issue in firebase-js-sdk.

Reply:
Done.  Changed the PR number to the link to the issue https://github.com/firebase/firebase-js-sdk/issues/449.